### PR TITLE
Add ru localization for 'requires higher OS version'

### PR DESCRIPTION
**Pull request by timsutton**

### CODE REVIEW

File: code/apps/Managed Software Center/Managed Software Center/ru.lproj/Localizable.strings [hunk 1]
```diff
@@ -289,6 +289,9 @@
 /* Install Required action text */
 "Required" = "Обязательное";
 
+ /* Item requires a higher OS version text */
```
Comment (clburlison) [inline code/apps/Managed Software Center/Managed Software Center/ru.lproj/Localizable.strings]: not sure how picky you want to be by line 292 has a space before the first `/`

Comment (timsutton) [inline code/apps/Managed Software Center/Managed Software Center/ru.lproj/Localizable.strings]: Thanks, fixed 👍

**Comment by gregneagle** [comment]
Thanks for the effort, but this feature is still in development and the exact wording is in flux -- it's very likely this will have to be redone soon.

**Comment by timsutton** [comment]
Thanks for the heads-up, happy to help with these as needed!

**Pull request merged**